source/solvers/heat_transfer.cc

```diff
@@ -940,6 +940,33 @@ HeatTransfer<dim>::write_checkpoint()
       sol_set_transfer.push_back(&previous_solutions[i]);
     }
   solution_transfer->prepare_for_serialization(sol_set_transfer);
+
+  // Serialize error table
+  std::string prefix =
+    this->simulation_parameters.simulation_control.output_folder;
+  std::string suffix = ".checkpoint";
+  if (this->simulation_parameters.analytical_solution->calculate_error())
+    serialize_table(
+      this->error_table,
+      prefix + this->simulation_parameters.analytical_solution->get_filename() +
+        "_HT" + suffix);
+  if (this->simulation_parameters.post_processing.calculate_heat_flux)
+    {
+      serialize_table(
+        this->heat_flux_table,
+        prefix +
+          this->simulation_parameters.post_processing.heat_flux_output_name +
+          suffix);
+    }
+  if (this->simulation_parameters.post_processing
+        .calculate_temperature_statistics)
+    {
+      serialize_table(
+        this->statistics_table,
+        prefix +
+          this->simulation_parameters.post_processing.temperature_output_name +
+          suffix);
+    }
```

Comment:
Is it intended that there are not `{}` for the first if, but that there are for the other two?